plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcColumnHandle.java

```diff
@@ -152,6 +152,15 @@ public long getRetainedSizeInBytes()
                 + jdbcTypeHandle.getRetainedSizeInBytes();
     }
 
+    public static JdbcColumnHandle createJdbcColumnHandle(String columnName, JdbcTypeHandle jdbcTypeHandle, Type columnType)
```

Comment:
> Refactor to always use JdbcColumnHandle.Builder instead of a constructor

`createJdbcColumnHandle` method defeats the purpose of this commit. What is the difference between factory method and constructor? I would inline this method.